investigate.py

```diff
@@ -539,28 +565,38 @@ def xforce_domain(target):
         )
 
 
-def ip_check(target):
+def ip_check(target, platforms):
     """
     Collection of all IP check functions to run.
     Comment out specific ones if you do not have an API key, as necessary.
```

Comment:
Missed this comment here and in `domain_check` too, should be removed

Reply:
Yep, thanks for the callout here.  I will do a quick update with this and a few other minor changes.